roles/kubelet/tasks/main.yml

```diff
@@ -39,39 +39,16 @@
     - "{{ cluster_config_path }}/pki/master/ca.pem"
 
 - name: Create kubelet kubeconfig
-  command: "kubectl config set-cluster kubernetes --certificate-authority {{ cluster_config_path }}/pki/master/ca.pem --embed-certs=true --server=https://{{ cluster_hostname }}:{{ cluster_port }} --kubeconfig {{ cluster_config_path }}/system:node:{{ hostvars[item].inventory_hostname }}.kubeconfig"
-  delegate_to: 127.0.0.1
-  run_once: True
-  with_items: "{{ groups['nodes'] }}"
```

Comment:
I haven't tried the latest commit, but now when we don't have `with_items` anymore, shouldn't `{{ hostvars[item].inventory_hostname }}`  be replaced with `{{ inventory_hostname }}`?